### DISCUSSION TITLE
Amazfit bip 3

### DISCUSSION THREAD

**Discussion by LOSEARDES77** [NightscoutFoundation/xDrip#3132] [Q&A]
Hi,
I recently bought an amazfit bip 3 pro and i was wondering if there were any possibilities to display bg to it.
I've read that bip 3 and bip 3 pro are about the same related to watchfaces.
Thanks!

**Comment by Navid200** [comment]
Have you asked watchdrip?

**Reply by LOSEARDES77** [reply]
They dont support it yet
